.travis_scripts/cross-test.sh

```diff
@@ -5,13 +5,13 @@ set -x
 
 # If no special target provided, use default cargo arch for building and run all unit tests
 if [ -z "$TARGET" ]; then
-    IRONCORE_ENV=dev cargo t --features "blocking beta" --verbose
+    IRONCORE_ENV=dev cargo test --features "blocking beta" --verbose
```

Comment:
Was there a reason to change this?

Reply:
Craig and Clint said `cargo t` used to have special meaning, but it doesn't anymore. It seemed like a good idea to use the long form of the command in CI. If that's not true, I'll change it back.

Reply:
Ya, that's sort of what I remembered as well which is why I called it out. I agree that if it doesn't have special meaning changing it to use the fully qualified command is better.